web/modules/custom/dpl_example_content/dpl_example_content.info.yml

```diff
@@ -56,3 +56,7 @@ default_content:
     - 0502836d-0f7b-4b21-9371-c40ae9e01495
     - 0fa7245a-3434-4d96-9793-a1848a22d37a
     - 807e6053-86ed-4dce-b6a9-a48b566b5910
+  menu_link_content:
+    - ff6db3ce-49e0-4107-b283-3403e22c5e6d
+    - c391079e-a4f5-4d29-992f-92ed1a2dd96c
+    - a25ebb74-e87f-4b2c-8156-b622dc23844e
```

Comment:
Consider adding all the elements from Figma to the default content.

This makes it easy for us to replicate what is currently the business reference point.

Reply:
Good point, done 👍